uptane/common.py

```diff
@@ -27,6 +44,17 @@ def sign_signable(signable, keys_to_sign_with):
     keys_to_sign_with:
       A list whose elements must conform to tuf.formats.ANYKEY_SCHEMA.
 
+    datatype:
```

Comment:
'json' and 'der', as the comment says (:

At first, I added a tuf.conf value SUPPORTED_METADATA_FORMATS = ['json', 'der'], but then pulled that out.... Think I should?